common/GoogleAnalytics.re

```diff
@@ -0,0 +1,16 @@
+let initTracking = [%raw
+  {|
+  function() {
+    var script=document.createElement('script');
```

Comment:
@ryyppy if you have time today or tomorrow we could convert this to reason. Not sure it's worth the trouble.